src/svg/qsvggenerator.cpp

```diff
@@ -166,6 +169,8 @@ class QSvgPaintEngine : public QPaintEngine
                        svgEngineFeatures())
     {
     }
+	int clip_counter = 0;
```

Comment:
I'm not sure this is the right way to initialise instance variables in C++.  That should be done in the constructors initializer list.

Reply:
It is okay since c++11. https://en.cppreference.com/w/cpp/language/data_members